gow.go

```diff
@@ -56,6 +55,7 @@ Options:
 	-s	Soft-clear terminal, keeping scrollback
 	-e	Extensions to watch, comma-separated; default: "go,mod"
 	-i	Ignored paths, relative to CWD, comma-separated
+	-g  Go compiler binary name
```

Comment:
_Super_ minor quibble: the spaces between flag and description should be a tab, otherwise it might render inconsistently. 🙂